builtin/providers/google/resource_compute_instance_group_manager.go

```diff
@@ -165,6 +207,22 @@ func resourceComputeInstanceGroupManagerRead(d *schema.ResourceData, meta interf
 	d.Set("target_size", manager.TargetSize)
 	d.Set("self_link", manager.SelfLink)
 
+	// Prepare the list of named ports
+	namedPortsCount := d.Get("named_port.#").(int)
+
+	namedPorts := make([]map[string]interface{}, 0, 1)
```

Comment:
I've not encountered issues with it yet, and I've used this branch to build instance_group_managers on 2 different GCE projects (in some tests with up to 10 named-ports). The order which seems to matter is within the .tf files themselves. I've repeatedly run "terraform apply" after the initial run and never seen anything being re-ordered.

Is this what you're asking?